flink-architecture-tests/pom.xml

```diff
@@ -137,13 +137,6 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
-		<dependency>
-			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-connector-elasticsearch5</artifactId>
```

Comment:
why?

Reply:
9649f45dab374c5b5a18ba12cc0526b37a504fd8 removed flink-connector-elasticsearch5 module, so it's not needed now.